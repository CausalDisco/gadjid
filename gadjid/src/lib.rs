// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
