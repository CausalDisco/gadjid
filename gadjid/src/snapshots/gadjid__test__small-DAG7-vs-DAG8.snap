---
source: src/lib.rs
expression: "test(\"20007.DAG-10\", \"20008.DAG-10\")"
---
g_true: 20007.DAG-10
g_guess: 20008.DAG-10
ts:
  - 8
  - 8
  - 0
  - 5
  - 7
ys:
  - 2
  - 0
  - 8
  - 8
  - 8
zs:
  - - 0
    - 1
    - 2
    - 3
    - 5
    - 6
    - 7
    - 9
  - - 0
    - 1
    - 2
    - 3
    - 5
    - 6
    - 7
    - 9
  - - 3
  - - 0
    - 2
    - 3
  - - 0
    - 2
    - 3
    - 5
    - 6
    - 9
nams:
  - []
  - []
  - []
  - []
  - []
nvas:
  - - 0
    - 1
    - 2
    - 3
    - 5
    - 6
    - 7
    - 9
  - - 0
    - 1
    - 2
    - 3
    - 5
    - 6
    - 7
    - 9
  - - 3
  - - 0
    - 2
    - 3
  - - 0
    - 2
    - 3
    - 5
    - 6
    - 9
pa_true_1st_T: []
an_true_1st_T:
  - 8
ch_true_1st_T: []
de_true_1st_T:
  - 8
poss_de_true_1st_T:
  - 8
prop_an_true_1st_T_and_1st_Y:
  - 2
shd:
  - 1
  - 45
oset_aid:
  - 0
  - 0
parent_aid:
  - 0
  - 0
ancestor_aid:
  - 0
  - 0
